src/Client.php

```diff
@@ -19,22 +19,43 @@ public function __construct(
     }
 
 
-    public function getContainer($idOrContainer)
+    public function getContainer($idOrContainer): Container
     {
+        $id = substr((string) $idOrContainer, -1, 1) === '/' ? (string)$idOrContainer : $idOrContainer . '/';
```

Comment:
substr() will implicitly coerce to a string for types that implement ```__toString()```, btw, so you don't need the (string) cast.